csharp/src/Microsoft.ML.OnnxRuntime/Training/TrainingSession.shared.cs

```diff
@@ -33,7 +33,16 @@ enum LRScheduler
     }
 
     /// <summary>
-    /// Represents a Training Session on an ONNX Model.
+    /// Trainer class that provides training, evaluation and optimizer methods for training an ONNX models.
+    ///
+    /// The training session requires four training artifacts
+    /// - The training onnx model
+    /// - The evaluation onnx model (optional)
+    /// - The optimizer onnx model
+    /// - The checkpoint directory
+    ///
+    /// These artifacts can be generated using the `onnxruntime-training` python [utility](https://github.com/microsoft/onnxruntime/blob/main/orttraining/orttraining/python/training/onnxblock/README.md).
+    ///
     /// This is a IDisposable class and it must be disposed of
     /// using either a explicit call to Dispose() method or
     /// a pattern of using() block. If this is a member of another
```

Comment:
/// A pointer to a underlying native instance of OrtTrainingSession

--> 

 /// A pointer to an underlying native instance of OrtTrainingSession